packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -112,10 +114,30 @@ contract OPContractsManager is ISemver {
         address mipsImpl;
     }
 
+    struct AddGameInput {
```

Comment:
I'm not sure about making this super configurable - seems like it makes it easier to misconfigure.  Maybe a middle-ground approach would be to treat these values as overrides, and use the values from the existing permissioned game if they are empty.